### PR TITLE
add fetch and style the cards a bit more

### PR DESCRIPTION
**Pull request by AlexF090**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,20 +1,43 @@
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import Header from './components/header';
-import Card from './components/card';
 import Navbar from './components/navbar';
 import styled from 'styled-components';
+import Card from './components/card';
+
+// const Card = ({ img, name }) => {
```
Comment (ThomasF85) [inline src/App.jsx:8]: unused code makes your files bigger and harder to read.

File: src/components/card.jsx [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
-    </CardStyle>
+    <CardList>
+      <CardImg src={img} alt={`Profile Picture ${name}`} />
+      <CardName>{name}</CardName>
+      <CardButton>show more</CardButton>
+    </CardList>
   );
 };
 
+const CardList = styled.li`
```
Comment (ThomasF85) [inline src/components/card.jsx:13]: This doesn't function as a CardList, but rather as a card container. The name is misleading.

**Review by ThomasF85 — APPROVED** [review]
Good job! You could still do a renaming.

File: src/App.jsx (hunk 1 above)
Comment (AlexF090) [reply src/App.jsx:8]: Thank you so much for your contructive comment!
I have changed it.

File: src/components/card.jsx (hunk 2 above)
Comment (AlexF090) [reply src/components/card.jsx:13]: Thank you so much for your contructive comment!
I have changed it.

**Pull request merged**